test/ppca.jl

```diff
@@ -0,0 +1,143 @@
+using MultivariateStats
+using Base.Test
+
+srand(34568)
+
+## PCA with zero mean
+
+X = randn(5, 10)
+Y = randn(3, 10)
+
+W = qr(randn(5, 5))[1][:, 1:3]
+σ² = 0.1
+M = PPCA(Float64[], W, σ²)
+
+@test indim(M) == 5
+@test outdim(M) == 3
+@test mean(M) == zeros(5)
+@test loadings(M) == W
+@test var(M) == σ²
+
+T = inv(W'*W .+ σ²*eye(3))*W'
+@test_approx_eq transform(M, X[:,1]) T * X[:,1]
```

Comment:
I think it would be proper to drop an old version support and rewrite everything when 0.6 will be release. New commands would require Compat bump which would be better to do in a separate PR.